force_gromacs/data_sources/simulation/simulation_model.py

```diff
@@ -25,6 +25,9 @@ class SimulationDataSourceModel(BaseDataSourceModel):
     #: Total number of fragments in simulation
     size = Int(1000)
 
+    #: Whether or not to overwrite existing simulation data
+    ow_data = Bool(False)
```

Comment:
Does the user see "Ow data" as the label in the UI? Shall we add a more informative label then, like "Overwrite simulation data"?

Reply:
That's a good idea, the `ow_data` attribute should already be included in the traits view, but I haven't added a custom label for it yet.